apps/src/pythonlab/pyodideWebWorker.js

```diff
@@ -12,11 +12,17 @@ async function loadPyodideAndPackages() {
   });
 }
 
-const pyodideReadyPromise = loadPyodideAndPackages();
+let pyodideReadyPromise = null;
+export async function initializePyodide() {
```

Comment:
Was there a particular reason for exporting?

Reply:
nope I'll fix, that was me working too quickly